components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -746,7 +746,7 @@ <h1 class="can-collapse closed" data-name="tags">
             <!-- MAP ANNOTATIONS -->
             <div id="mapAnnsPane">
             <h1 class="can-collapse closed" data-name="keyvaluepairs">
-                Key-Value Pairs
+                Attributes
```

Comment:
@dominikl @gusferguson any objection on that change as it seams to be more appropriate one. I took it from how @eleanorwilliams called that panel.